services/usb-livesync-service-base.ts

```diff
@@ -48,7 +48,8 @@ export class UsbLiveSyncServiceBase implements IUsbLiveSyncServiceBase {
 		beforeBatchLiveSyncAction?: (_filePath: string) => IFuture<string>,
 		iOSSimulatorRelativeToProjectBasePathAction?: (projectFile: string) => string): IFuture<void> {
 		return (() => {
-			let synciOSSimulator = this.$hostInfo.isDarwin ? this.$iOSEmulatorServices.isSimulatorRunning().wait() || (this.$options.emulator && platform.toLowerCase() === "ios") : false;
+			let platformLowerCase = platform.toLowerCase();
+			let synciOSSimulator = this.$hostInfo.isDarwin ? platformLowerCase !== "android" && (this.$iOSEmulatorServices.isSimulatorRunning().wait() || (this.$options.emulator && platformLowerCase === "ios")) : false;
```

Comment:
`hostInfo.isDarwin` - is a property - https://github.com/telerik/mobile-cli-lib/blob/master/declarations.d.ts#L338 not a function